### PR TITLE
[10.0][MIG] Migration of base_mail_bcc

### PR DESCRIPTION
**Pull request by fkantelberg**
Hello,
because we need this module in odoo 10 I want to share the migrated version

### CODE REVIEW

File: base_mail_bcc/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (tbaden) [inline base_mail_bcc/README.rst]: please use the new readme generator (https://github.com/OCA/maintainer-tools)

File: base_mail_bcc/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Thomas Rehn (initOS GmbH)
```
Comment (tbaden) [inline base_mail_bcc/__init__.py]: copyright s/2014-2016/2014-2019/ in all files

File: base_mail_bcc/models/ir_mail_server.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            config_email_bcc = config_email_bcc.encode('ascii')
+            existing_bcc = []
+            if message['Bcc']:
+                existing_bcc.append(message['Bcc'])
+                del message['Bcc']
+            message['Bcc'] = COMMASPACE.join(
+                existing_bcc + config_email_bcc.split(',')
+            )
+
+        return super(IrMailServer, self).send_email(
```
Comment (tbaden) [inline base_mail_bcc/models/ir_mail_server.py:36]: possible readability improvement (not mandatory)

```
return super(IrMailServer, self).send_email(
            message, 
            mail_server_id=mail_server_id,
            smtp_server=smtp_server,
            smtp_port=smtp_port,
            smtp_user=smtp_user,
            smtp_password=smtp_password,
            smtp_encryption=smtp_encryption,
            smtp_debug=smtp_debug
        )
```

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
i did a code review.
just some minor details

**Comment by fkantelberg** [comment]
@tbaden Thanks for the review. I did the mentioned changes.

**Review by tbaden — APPROVED** [review]
thanks

**Review by tbaden — CHANGES_REQUESTED** [review]
please check travis and runbot
runbot:
WARNING: module base_mail_bcc: description is empty !
travis:
ERROR openerp_test odoo.addons.mail_sendgrid.tests.test_mail_sendgrid: FAIL

**Comment by hbrunn** [comment]
please rebase

**Pull request closed without merging**